### DISCUSSION TITLE
Changing connection name or host

### DISCUSSION THREAD

**Discussion by NullpointerForever** [dbeaver/dbeaver#153] [Q&A] [answered]
Hi,

After change a connection name or host in connection and restart DBeaver - settings is not stored. I don't known what about other settings.

Regards

**Comment by codebling** [comment]
I can't reproduce this. DBeaver settings are stored in `~/.dbeaver`, is it possible that the permissions are set such that DBeaver cannot modify the files there?

**Comment by NullpointerForever** [comment, marked as answer]
I have Windows 10. After disabling hidden attribute from files in .dbeaver dir, it's OK. This attribute hidden was from copying this directory from previous Windows installation (I suppose). Thanks for help.

**Comment by codebling** [comment]
Great !
